.github/workflows/bindings_python.yml

```diff
@@ -0,0 +1,273 @@
+            target: x86
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+          architecture: ${{ matrix.platform.target }}
+      - name: Build wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.platform.target }}
+          args: --release --find-interpreter
+          sccache: 'true'
+          working-directory: bindings/imap-codec-python
+      - name: Upload wheels
+        uses: actions/upload-artifact@v4
+        with:
+          name: wheels-windows-${{ matrix.platform.target }}
+          path: target/wheels
+      - name: pytest
+        if: ${{ !startsWith(matrix.platform.target, 'aarch64') }}
+        shell: bash
+        run: |
+          set -e
+          cd bindings/imap-codec-python
+          python3 -m venv venv
+          source venv/Scripts/activate
+          pip install imap-codec --find-links ../../target/wheels --force-reinstall
+          pip install pytest
+          pytest
+
+  macos:
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: macos-12
+            target: x86_64
+          - runner: macos-14
+            target: aarch64
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.x
+      - name: Build wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.platform.target }}
+          args: --release --find-interpreter
+          sccache: 'true'
+          working-directory: bindings/imap-codec-python
+      - name: Upload wheels
+        uses: actions/upload-artifact@v4
+        with:
+          name: wheels-macos-${{ matrix.platform.target }}
+          path: target/wheels
+      - name: pytest
+        run: |
+          set -e
+          cd bindings/imap-codec-python
+          python3 -m venv venv
+          source venv/bin/activate
+          pip install imap-codec --find-links ../../target/wheels --force-reinstall
+          pip install pytest
+          pytest
+
+  sdist:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Build sdist
+        uses: PyO3/maturin-action@v1
+        with:
+          command: sdist
+          working-directory: bindings/imap-codec-python
+      - name: Upload sdist
+        uses: actions/upload-artifact@v4
+        with:
+          name: wheels-sdist
+          path: target/wheels
+
+  release:
+    name: Release package to PyPI
+    runs-on: ubuntu-latest
+    if: ${{ startsWith(github.ref, 'refs/tags/imap-codec-python/v') }}
+    needs: [linux, musllinux, windows, macos, sdist]
+    steps:
+      - uses: actions/checkout@v4
+      - name: Assert release version matches package version
+        run: |
+          set -e
+
+          # Get release version from Git tag
+          tag_version=${GITHUB_REF#refs/tags/imap-codec-python/v}
```

Comment:
Oh, nice trick! This matches `x.y.z`, right?

Reply:
Exactly, the shell parameter expansion removes the given pattern from the beginning of the value.